torch/csrc/distributed/rpc/ProcessGroupAgent.h

```diff
@@ -42,7 +39,8 @@ class ProcessGroupAgent : public RpcAgent {
 
  private:
   // put SendWork into a queue and notify the sendLoop thread
-  void enqueue(SendWork work);
+  void enqueueSend(RpcWork work);
+  void enqueueRecv(RpcWork work);
   // sending out the message
   void sendLoop();
```

Comment:
Didn't we remove `sendLoop()` from the cpp file? We do we still have the definition?

Reply:
Yes, this is removed in my local version.